src/Frontend/Core/Engine/Form.php

```diff
@@ -194,6 +196,7 @@ public function addDate(
      * @param bool   $multipleSelection Is it possible to select multiple items?
      * @param string $class             Class(es) that will be applied on the element.
      * @param string $classError        Class(es) that will be applied on the element when an error occurs.
+     *
      * @return \SpoonFormDropdown
      */
     public function addDropdown(
```

Comment:
they are extending the spoon stuff and need to keep the signature of the parent class